content/docs/1.1.0/volumes-and-nodes/maintenance.md

```diff
@@ -17,20 +17,19 @@ Currently, it's recommended to shut down the workloads with Longhorn volume befo
 If shutting down the workloads is not possible, follow the steps below to minimize the impact for node maintenance:
 
 1. Cordon the node. Longhorn will automatically disable the node scheduling when a Kubernetes node is cordoned.
+	Make sure the node has been disabled from scheduling, then evict all the replicas on the node.
 
-1. Drain the node to move the workload to somewhere else.
+2. Drain the node to move the workload to somewhere else.
 
     You will need to use `--ignore-daemonsets` and `--force` options to drain the node.
 
     The replica processes on the node will be stopped at this stage. 
         
-    > **Upcoming feature:** After adding the support of `Replica eviction`, you will be able to evict the replicas on the node gracefully.
-    
     The engine processes on the node will be migrated with the Pod to other nodes.
 
     After the `drain` is completed, there should be no engine or replica process running on the node. Two instance managers will still be running on the node, but they're stateless and won't cause interruption to the existing workload.
-1. Perform the necessary maintenance, including shutting down or rebooting the node.
-1. Uncordon the node. Longhorn will automatically re-enable the node scheduling.
+3. Perform the necessary maintenance, including shutting down or rebooting the node.
```

Comment:
We can change back to use `1.` for all entries so they can be automatically ordered when rendering.

Reply:
Good to know. Fixed. Thanks.